sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClient.cs

```diff
@@ -72,19 +74,14 @@ public ContainerRegistryClient(Uri endpoint, TokenCredential credential, Contain
             Argument.AssertNotNull(credential, nameof(credential));
             Argument.AssertNotNull(options, nameof(options));
 
-            if (options.Audience == null)
```

Comment:
Was this done for consistency across languages? Is this a breaking change then?

Reply:
It's not a breaking change to remove an exception.  No client in the wild could be functioning without passing the Audience right now, so setting a default will not change any existing behavior.

@lmolkova confirmed with the service team that this is the correct path forward.